### ISSUE TITLE
Order status after capture setting

### ISSUE THREAD

**Issue by Feri-6** [paylike/plugin-prestashop-1.6#17]
What does the "Order status after capture" setting mean? We sell real (not virtual) products, but sometimes something is not on stock. What is the correct setting for us?

**Comment by ionutcalara** [comment]
@Feri-6 when you capture the funds, based on your settings, you might capture funds right away (Instant) or when the product get's delivered (delayed). In your case I think it should be delayed since you sell real products. You should capture only after you ship. One option could be: "Shipped" or if you capture after you get shipping confirmation, then it can be "Delivered". This is up to you, and what works best. 

It's important to note, that when you are selling real products, Capture mode should be set to delayed, and you need to capture after the product has been shipped. This decreases the chances that you need to do a refund, due to how the payment system works. 

So to recap, when the users pays via the popup, the funds are "Authorized". There's a hold on them at the bank, but no transfer takes place. At this time, if something is wrong, the funds can be voided, and that just removes the hold. This means there's no refund. 

When you capture - via the tool on the order screen, the funds are transferred, and if there are issues you will need to do a refund.

**Comment by Feri-6** [comment]
Thanks. And what can we do if something is not on stock or the client wants to add something to his/her order? So how can we modify the captured sum?

**Comment by ionutcalara** [comment]
You can't capture more than what the client has authorized for security reasons, they need to explicitly allow a certain sum and they do that via the popup. In your case, you should cancel the order and have the user create a new one.
